pkgs/development/interpreters/luajit/2.0.nix

```diff
@@ -1,13 +1,13 @@
 { self, callPackage, fetchFromGitHub, lib, passthruFun }:
 
-callPackage ./default.nix {
-  version = "2.0.5-2022-09-13";
+callPackage ./default.nix rec {
+  version = "2.0.${lib.fileContents ${src}/.relver}";
```

Comment:
Please, no import-from-derivation in nixpkgs.  `src` is a derivation that needs to be built.  The `version` attribute must not depend on it.

Reply:
And it fails for me anyway
```
error: syntax error, unexpected DOLLAR_CURLY, expecting '}'
```

Reply:
Sorry about that, looks like I haven't pushed all the changes.

Ok, I don't have better any idea than hard-coding git commit timestamp.